templates/compositions/ec-component-navigation/navigation-list.html.twig

```diff
@@ -0,0 +1,53 @@
+{% spaceless %}
```

Comment:
Why is this template part of this PR? It doesn;t seem to involve any of the multilingual components at all? At the same time, the navigation list composition described in the ECL is used to navigate a page, not to add shortcuts to different functionalities so I'm not sure its the best composition to use.

Reply:
I added this template to unblock us when we want to use the link 'edit' of a node (used in Behat tests) or others links. I didn't see any ticket concerning this requirement.
It's not a design component, it's a pure drupal menu. I tried to put it very simple, only to unblock us. But I can remove this template and wait for ecl-twig to create one, as you wish.

Reply:
Removed and PR created https://github.com/openeuropa/oe_theme/pull/319